tests/database.js

```diff
@@ -1,8 +1,19 @@
 import test from 'ava';
-import db from '../lib/database';
+import config from 'config';
+import _ from 'lodash';
+
+import database from '../lib/database';
 
 test('Database Initialization', t => {
-  return db.init().then(() => {
+  return database.init().then(() => {
     t.pass();
+  }).catch(e => {
+    if (_.includes(e.message, `database "${config.knex.connection.database}" does not exist`)) {
+      // warning added for safety
+      console.log('--------------------------------------------------------------------------'); // eslint-disable-line no-console
```

Comment:
Can we keep it? Huh? Can we?
This need for a separate database is new. I just thought I'd gussy up the warning.

Changes desired? Or no warning at all?